corehq/motech/openmrs/openmrs_config.py

```diff
@@ -184,7 +184,10 @@ class ObservationMapping(DocumentSchema):
         }
 
     """
-    concept = StringProperty()
+    # If no concept is specified, this ObservationMapping is used for
+    # setting a case property or creating an extension case for any
+    # concept
+    concept = StringProperty(required=False, default=None)
```

Comment:
I agree -- `None` should not be a special value. ... In fact I went with "all" and made it a constant. It can't clash with a UUID, and it makes the meaning more obvious.